ofrak_core/ofrak/gui/server.py

```diff
@@ -260,6 +264,68 @@ async def run_until_cancelled(self):  # pragma: no cover
         finally:
             await self.runner.cleanup()
 
+    @exceptions_to_http(SerializedError)
+    async def init_chunked_root_resource(self, request: Request) -> Response:
+        name = request.query.get("name")
+        size_param = request.query.get("size")
+        size = int(size_param) if size_param is not None else None
+        if name is None:
+            raise HTTPBadRequest(reason="Missing resource name from request")
+        if size is None:
+            raise HTTPBadRequest(reason="Missing chunk size from request")
```

Comment:
Minor, but I think this is a little easier to read. May also make mypy happier.

```suggestion
        if name is None:
            raise HTTPBadRequest(reason="Missing resource name from request")
        if size_param is None:
            raise HTTPBadRequest(reason="Missing chunk size from request")
        size = int(size_param)
```